### PR TITLE
Changed if tags to if filters

### PR DESCRIPTION
**Pull request by shaidar**

### CODE REVIEW

File: extensions/_modules/boto_ec2.py [hunk 1]
```diff
@@ -1813,7 +1813,7 @@ def detach_volume(volume_id=None, instance_id=None, device=None, force=False,
         except boto.exception.BotoServerError as e:
             log.error(e)
             return False
-    elif tags:
+    elif filters:
         try:
             volumes = conn.get_all_volumes(filters = {'tag:{0}'.format(tag_name):'{0}'.format(tag_value)})
```
Comment (blarghmatey) [inline extensions/_modules/boto_ec2.py]: Where are tag_name and tag_value coming from?

Comment (shaidar) [reply extensions/_modules/boto_ec2.py]: They are part of salt-call:
`salt-call boto_ec2.detach_volume filters='{"tag:<tag_name>":"<tag_value>"}'`

Comment (blarghmatey) [reply extensions/_modules/boto_ec2.py]: The `tag_name` and `tag_value` variables aren't getting populated anywhere in the function. Did you mean to turn that into a dictionary comprehension?

**Comment by blarghmatey** [comment]
👍

**Pull request merged**